packages/manager/modules/pci/src/projects/project/storages/containers/add/components/associate-user-to-container/componenets/credential-banner/index.js

```diff
@@ -0,0 +1,15 @@
+import angular from 'angular';
+import 'angular-translate';
+import '@ovh-ux/ui-kit';
+
+import component from './component';
+
+const moduleName =
+  'ovhManagerPciStoragesContainersAddCreateLinkedUserCredentialBanner';
+
+angular
+  .module(moduleName, ['pascalprecht.translate', 'oui'])
+  .component('userCredentialBanner', component)
```

Comment:
**issue(blocking)**: Component must be prefixed (prevent using too generic name for component).